tools/ci_build/github/azure-pipelines/nuget-cuda-publishing-pipeline.yml

```diff
@@ -1,7 +1,7 @@
 resources:
   pipelines:
   - pipeline: build
-    source: 'Nuget-CUDA-Packaging-Pipeline'
+    source: 'CUDA-Zip-Nuget-Java-Packaging-Pipeline'
```

Comment:
How about add "12" there?

Reply:
This pipeline can also build Cuda 11. In the future it should be able to build Cuda 13+ with some minor modifications.